packages/esm-reports-app/src/components/scheduled-overview.component.tsx

```diff
@@ -13,11 +12,12 @@ import {
   TableHeader,
   TableRow,
 } from '@carbon/react';
+import { ExtensionSlot, isDesktop, useLayoutType, usePagination } from '@openmrs/esm-framework';
 import { useScheduledReports } from './reports.resource';
 import { DEFAULT_PAGE_SIZE, DEFAULT_PAGE_SIZES } from './pagination-constants';
 import ScheduledOverviewCellContent from './scheduled-overview-cell-content.component';
 import Overlay from './overlay.component';
```

Comment:
Ditto

Reply:
Cries in "We should have an ESLint rule for this already"